shared/chat/conversation/participant-rekey.native.js

```diff
@@ -36,7 +36,6 @@ const containerStyle = {
   ...globalStyles.flexBoxColumn,
   alignItems: 'stretch',
   backgroundColor: globalColors.darkBlue4,
-  borderLeft: `1px solid ${globalColors.black_20}`,
```

Comment:
just a quick note about this. we can have cross-system styles if we just break it out. so if you use borderLeftWidth: 1 and borderStyle: 'solid' borderRightWith: 0 borderLeftColor: globalColors.black_20 it'll work on desktop as well